iroha/src/wsv.rs

```diff
@@ -1,19 +1,21 @@
 //! This module provides `WorldStateView` - in-memory representations of the current blockchain
 //! state.
 
-use std::collections::HashSet;
+use std::collections::HashMap;
 
-use crate::prelude::*;
 use config::Configuration;
 use iroha_data_model::prelude::*;
 
+use crate::prelude::*;
+use crate::tx::RejectedTransaction;
+
 /// Current state of the blockchain alligned with `Iroha` module.
 #[derive(Debug, Clone)]
 pub struct WorldStateView {
     /// The world - contains `domains`, `triggers`, etc..
     pub world: World,
     /// Hashes of the committed and rejected transactions.
-    pub transactions_hashes: HashSet<Hash>,
+    pub transactions: HashMap<Hash, FinalizedTransaction>,
```

Comment:
Hi!

We have a world for storing transactions, no need to add additional information in world state view.
If we need to store rejected transactions (which is quite tricky and should be discussed with @eadventurous and @takemiyamakoto ) - then they also should be placed inside world and blocks.

Reply:
Hi! Thank you for response. 

We will discuss it with team

Reply:
We have decided to create a separate task to retrieve transactions history from the world due it require a lot of efforts and do not fit in this PR